File: C#/Dal/filter.cs

using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Dal
{
    public class Filter
    {
        public Kashrut[] kashrut { get; set; }
        public Invited[] invitds { get; set; }
        public Area[] areas { get; set; }
        public HallType[] hallTypes { get; set; }
        public int price { get; set; }

    }
}

File: Angular/src/app/models/filter.ts
import Area from "./AreaDto";
import HallType from "./hallTypeDto";
import Invited from "./InvitedDto";
import Kashrut from "./KashrutDto";

export default class Filter {
areas:Area[] = []
invitds: Invited[] = []
kashrut:Kashrut[]=[]
hallTypes:HallType[]=[]
price:number=0; 
}